ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -209,6 +209,19 @@ def run_healthcheck(host, source=None, check=None, output_type="json",
     return result.returncode, data
 
 
+def set_excludes(host):
+    """Mark checks that should be excluded from the results"""
+    conf = host.get_file_contents(
+        '/etc/ipahealthcheck/ipahealthcheck.conf', encoding='utf-8'
+    )
+    conf += '[excludes]'
```

Comment:
If you wanted to check, if the section is already there and avoid failure, this would also work
```
    config_data = host.get_file_contents(config, encoding='utf-8')
    cfg = RawConfigParser()
    cfg.read_string(config_data)
    # add section "excludes" if it doesn't exist
    excludes_section = "excludes"
    if not cfg.has_section(excludes_section):
        cfg.add_section(excludes_section)
    # add options to the excludes section
    for option, value in excluded_checks:
        if not cfg.has_option(excludes_section, option):
            cfg.set(excludes_section, option, value)
# output to the original config file
```

Reply:
What's strange is this code worked over multiple runs yesterday, just not the last one I ran for the day.

Reply:
Without `update_packages=true`, the 389-ds version is the one from the vagrant image and it was generated a long time ago (with 389-ds-base-2.3.2-2). With `update-packages=true` the test downloads the latest version from the updates repo (2.3.7-2), and the check for log buffering was added in 2.3.5.